### PR TITLE
Update Ruby + Dependencies

### PR DESCRIPTION
**Pull request by wallin**
* Bump dependencies
* Drop outdated Ruby versions
* Replace Travis with GH Actions

### CODE REVIEW

File: .github/workflows/specs.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.7.7, 3.1.3]
```
Comment (opti) [inline .github/workflows/specs.yml]:
```suggestion
        ruby-version: [2.7, 3.1]
```
This way it's easier to stay on the latest versions.

File: .github/workflows/specs.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby ${{ matrix.ruby-version }}
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+    - name: Install dependencies
+      run: |
+        gem install bundler
```
Comment (opti) [inline .github/workflows/specs.yml]: bundler is part of ruby already, can skip this step. Though keeping it up to date could be more useful:
```suggestion
        gem update --system
```

**Review by opti — APPROVED** [review]

File: .github/workflows/specs.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.7, 3.1]
+    steps:
+    - uses: actions/checkout@v2
```
Comment (nijikon) [inline .github/workflows/specs.yml]: actions/checkout@v3

File: .github/workflows/specs.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    - name: Set up Ruby ${{ matrix.ruby-version }}
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby-version }}
+    - name: Install dependencies
+      run: |
+        gem update --system
```
Comment (nijikon) [inline .github/workflows/specs.yml:19]: Not sure you need this unless you want to be on the latest bundler on each ruby version.

**Pull request merged**